src/test/java/com/faforever/client/chat/AbstractChatTabControllerTest.java

```diff
@@ -194,7 +196,7 @@ public void testOnSendMessageSuccessful() throws Exception {
     instance.onSendMessage();
 
     verify(chatService).sendMessageInBackground(eq(receiver), eq(message));
-    assertThat(instance.messageTextField().getText(), is(emptyString()));
+    assertTrue(instance.messageTextField().getText().equals(""));
```

Comment:
Why are you changing this, did it not work?

Reply:
it does not work on linux using the OpenJDK.

Reply:
This has something to do with new mockito version, even though I don'd understand how it suddenly happened without breaking builds before.

New API is `isEmptyString()` AFAIK.